lib/oli_web/router.ex

```diff
@@ -789,15 +789,15 @@ defmodule OliWeb.Router do
   scope "/workspaces/", OliWeb.Workspaces do
     pipe_through([:browser, :authoring_and_delivery, :set_sidebar])
 
-    live_session :workspaces,
+    live_session :course_author_workspace,
```

Comment:
Since this session contains instructor and student views, I'm not sure it makes sense to rename it
to course_author_workspace

Reply:
Good catch! The `:course_author_workspace` name was a leftover from splitting the workspaces. Now that all workspaces are combined, it makes sense to revert back to `:workspaces`.